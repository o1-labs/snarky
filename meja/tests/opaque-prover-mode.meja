Prover {
  type t = opaque(int);
};
