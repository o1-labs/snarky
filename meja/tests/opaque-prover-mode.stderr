File "tests/opaque-prover-mode.meja", line 2, characters 11-22:
Error: The type opaque(int) is not valid in this mode:
       opaque types cannot be created in Prover mode.
